src/main/java/io/openmessaging/storage/dledger/DLedgerLeaderElector.java

```diff
@@ -163,6 +163,7 @@ public void changeRoleToLeader(long term) {
     public void changeRoleToCandidate(long term) {
         synchronized (memberState) {
             if (term >= memberState.currTerm()) {
+                lastParseResult = VoteResponse.ParseResult.WAIT_TO_REVOTE;
```

Comment:
lastParseResult should rely on the results of the last vote, such as the existence of a larger term in another peer, in this case, need to increase the term，so this status will set to WAIT_TO_REVOTE_NEXT